curp/src/cmd.rs

```diff
@@ -101,7 +101,7 @@ impl ConflictCheck for u32 {
 /// Command executor which actually executes the command.
 /// It is usually defined by the protocol user.
 #[async_trait]
-pub trait CommandExecutor<C>: Sync + Send + Clone + std::fmt::Debug
```

Comment:
What's the purpose of `Clone`. I thought ce is meant to be shared by `Arc` not cloning.

Reply:
So I removed `Clone`

Reply:
Sry.